content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -349,3 +350,106 @@ rules to reach the proper Istio service in the main cluster.
+    1.  control plane security enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+
+1.  Istio remote clusters deployed with
+    1.  control plane security enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+        1.  The CA certificate for the remote clusters needs to be signed by the same CA or root CA as the main cluster.
+    1.  Istio pilot service hostname resolvable via DNS
+        1.  Required because Istio configures the sidecar to verify the certificate subject names using the `istio-pilot.<namespace>` subject name format.
+    1.  Control plane IPs or resolvable host names set
+
+### mTLS between application pods
+
+The steps to enable mTLS for all application pods are as follows:
+
+1.  Istio control plane cluster deployed with
+    1.  Global mTLS enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+
+1.  Istio remote clusters deployed with
+    1.  Global mTLS enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+        1.  The CA certificate for the remote clusters needs to be signed by the same CA or root CA as the main cluster.
+
+**NOTE:**  The CA certificate steps are identical for both control plane security and application pod security steps.
+
+### Example deployment
+
+The following is an example procedure to install Istio with both control plane mTLS and application pod
+mTLS enabled.  The example sets up a remote cluster with a selector-less service and endpoint for `istio-pilot` to
+allow the remote sidecars to resolve `istio-pilot.istio-system` hostname via its local kubernetes DNS.
+
+1.  *Primary Cluster.*  Deployment of the Istio control plane cluster
+
+    1.  Create the `cacerts` secret from the Istio samples certificate in the `istio-system` namespace:
+
+        {{< text bash >}}
+        $ kubectl create ns istio-system
+        $ kubectl create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
+        {{< /text >}}
+
+    1.  Deploy the Istio control plane with control plane and application pod security enabled
+
+        {{< text bash >}}
+        $ helm template --namespace=istio-system \
```

Comment:
What about scenarios where the operator has already installed (`helm upgrade` can handle this case, as can `kubectl apply`)?  What about helm install?

Reply:
This is just an example to show the args used for the control-plane security scenario.  IMO, we shouldn't include examples all over the place of helm template v. upgrade v. helm install.  Explaining those details in the general install doc should be sufficient and not redoing it all over would keep the other sections more clear, concise, and consistent/maintainable.

Reply:
I agree the duplication in the documentation is a real problem but I'd rather not compound it.  Any chance you can rework this section to explain the helm flags needed rather than show the actual command?

Cheers
-steve

Reply:
I'm not really sure how to solve this problem other than holistically.  I guess we can merge it as is, and then rework it in followup PRs if your willing to do that once we come up with a good approach for handling all of the helm args duplication.